CefSharp/ModelBinding/ModelBindingExtensions.cs

```diff
@@ -4,16 +4,373 @@
 
 using System;
 using System.Collections.Generic;
+using System.Globalization;
 using System.Linq;
 using System.Reflection;
 
 namespace CefSharp.ModelBinding
 {
     /// <summary>
-    /// Containing extensions for the <see cref="Type"/> object.
+    /// Contains extension methods for data marshaling.
     /// </summary>
     internal static class ModelBindingExtensions
     {
+
+        /// <summary>
+        /// The custom parser below supports converting strings into a <see cref="Enum"/> that has the <see cref="FlagsAttribute"/> defined.
+        /// These are commonly used separators to assist that process.
+        /// </summary>
+        private static readonly char[] EnumSeparators = { '|', ',', ';', '+', ' ' };
```

Comment:
Do you have a reference source for this list? Are you using different separators in your code to represent a set of flags? 

Do we really need to support all these possibilities?